### PR TITLE
WIP https://issues.redhat.com/browse/ACM-3383--rearranging and editing, a…

### PR DESCRIPTION
**Pull request by swopebe**
…ws together, bm together, stronger intro

### CODE REVIEW

File: clusters/hosted_control_planes/managing_hosted_aws.adoc [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+* <<create-hosted-aws,Creating a hosted cluster on AWS with the console>>
+* <<importing-hosted-cluster-ui-aws,Importing a hosted control plane cluster on AWS with the console>>
+* <<importing-hosted-cluster-cli-aws,Importing a hosted control plane cluster on AWS with the CLI>>
+* <<hosting-service-cluster-access-aws,Accessing a hosting cluster on AWS>>
+* <<hypershift-cluster-destroy-aws,Destroying a hosted cluster on AWS>>
+
+[#hosted-prerequisites-aws]
+== Prerequisites
+
+You must configure hosted control planes before you can create hosted control plane clusters. See xref:../../clusters/hosted_control_planes/configure_hosted.adoc#hosted-control-planes-configure[Configuring hosted control planes (Technology Preview)] for more information.
```
Comment (lahinson) [inline clusters/hosted_control_planes/managing_hosted_aws.adoc]: I think this xref might need to be updated. Should it now point to configure_hosted_aws.adoc?

File: clusters/hosted_control_planes/disable_hosted.adoc [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [#disable-hosted-control-planes]
 = Disabling the hosted control plane feature
 
-When disabling the hosted control plane cluster feature, you must destroy the hosted cluster and the managed cluster resource on {mce-short}. See xref:../hosted_control_planes/delete_hosted.adoc#destroy-hosted-control-planes[Destroying hosted control plane clusters] for more information.
+You can uninstall HyperShift and disable the hosted control plane. When disabling the hosted control plane cluster feature, you must destroy the hosted cluster and the managed cluster resource on {mce-short}, as described in the _Managing hosted control plane clusters_ topics.
```
Comment (lahinson) [inline clusters/hosted_control_planes/disable_hosted.adoc]: Should this be "You can uninstall the HyperShift Operator and disable the hosted control plane"?

**Review by lahinson — APPROVED** [review]
I added a couple of minor comments, but overall, LGTM! Note that I'll be adding docs about configuring hosted control planes on KubeVirt for 2.8, so I'll follow this new file structure when I do that.

File: clusters/hosted_control_planes/disable_hosted.adoc (hunk 2 above)
Comment (swopebe) [reply clusters/hosted_control_planes/disable_hosted.adoc]: That works.

File: clusters/hosted_control_planes/managing_hosted_aws.adoc (hunk 1 above)
Comment (swopebe) [reply clusters/hosted_control_planes/managing_hosted_aws.adoc]: yes.

**Pull request merged**